iml-sfa.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=IML SFA Service
+PartOf=iml-manager.target
+After=postgresql-9.6.service
+After=iml-settings-populator.service
+Requires=iml-settings-populator.service
+
+
+[Service]
+Type=simple
+Environment=RUST_LOG=info
```

Comment:
I think we recently moved the RUST_LOG environment variable to the iml-settings.conf didn't we?

Reply:
actually, I was thinking of the overrides file. nvm.

Reply:
Not on the manager.